### PR TITLE
Add a new bid adapter and test specs for bridgewell

### PR DESCRIPTION
**Pull request by kuchunchou**
## Type of change
- [x] New bidder adapter   

## Description of change
This is a new bidder adapter pull request for Bridgewell (an buyer based in Taiwan).

- test parameters for validating bids
```
{
    sizes: [
        [300, 250]
    ],
    bids: [{
        bidder: 'bridgewell',
        params: {
            ChannelID: 'CgUxMjMzOBIBNiIFcGVubnkqCQisAhD6ARoBOQ',
        }
    }]
}
```

- contact email of the adapter’s maintainer: kuchunchou@bridgewell.com
- [x] official adapter submission

### CODE REVIEW

File: modules/bridgewellBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} validBidRequests - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */ 
+  buildRequests: function(validBidRequests) {
+    var channelIDs = [];
```
Comment (jaiminpanchal27) [inline modules/bridgewellBidAdapter.js]: Use `let/const`

File: modules/bridgewellBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          return;
+        } else if (typeof matchedResponse.ad !== 'string') {
+          return;
+        } else if (typeof matchedResponse.net_revenue === 'undefined') {
+          return;
+        } else if (typeof matchedResponse.currency !== 'string') {
+          return;
+        }
+
+        bidResponse.bidderCode = spec.code;
```
Comment (jaiminpanchal27) [inline modules/bridgewellBidAdapter.js]: `bidderCode` will be set by bidderFactory so you can remove this line

File: modules/bridgewellBidAdapter.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        bidResponse.currency = matchedResponse.currency;
+
+        bidResponses.push(bidResponse);
+      }
+    });
+
+    return bidResponses;
+  },
+
+  getUserSyncs: function(syncOptions) {
```
Comment (jaiminpanchal27) [inline modules/bridgewellBidAdapter.js]: You can remove this function if not needed. Its optional.

**Comment by kuchunchou** [comment]
@jaiminpanchal27 Thanks for the review. Unit tests should pass now and issues stated in your comments are also fixed accordingly.

**Pull request merged**